act-rules-format.bs

```diff
@@ -235,158 +257,198 @@ When all expectations are true for a test target, the test target `passed` the r
 </div>
 
+If any [accessibility requirements](#accessibility-requirements) are listed, the ACT Rule MUST indicate how each outcome impacts the conformance to the listed accessibility requirements. For WCAG 2.1 Success Criteria, this means indicating whether or not the success criterion can be satisfied, or if further testing is needed.
 
-Test Subject {#output-test-subject}
------------------------------------
-
-When a single URL can be used to reference the web page, or other test subject, this URL MAY be used. In scenarios where more complex actions are required to obtain the test subject (in the state that it is to be tested), it is left to ATT developers to determine which method is best used to express the test subject.
-
-
-Test Target {#output-test-target}
----------------------------------
-
-When representing the test target in the output data, it is often impractical or impossible to serialize the test target as a whole. Instead of this, a pointer can be used to indicate where the test target exists within the web page or other [test subject](#output-test-subject). There are a variety of pointer methods available, such as those defined in [Pointer Methods in RDF 1.0](https://www.w3.org/TR/Pointers-in-RDF/) [[Pointers-in-RDF]].
-
-The pointer method used in the output data of an ACT Rule MUST include the pointer method used in [Test cases](#test-cases).
-
-
-Outcome {#output-outcome}
--------------------------
-
-The definition of a rule MUST always result in one of the following outcomes:
-
-- **Passed**: All [expectations](#expectations) for the [Test Target](#output-test-target) were true
-- **Failed**: One or more expectations for the Test Target was false
-- **Inapplicable**: There were no Test Targets in the [Test Subject](#output-test-subject)
+<div class=example>
+  <p>Outcome of this rule map to the accessibility requirements in the following manner:</p>
+  <ul>
+    <li>`failed`: The success criteria are *not satisfied*</li>
+    <li>`passed` or `inapplicable`: Further testing is necessary to determine if the success criteria are satisfied</li>
+  </ul>
+</div>
 
-<div class="note">
-  While *inapplicable* is a valid result for ACT Rules, it may not be a valid result for all [accessibility requirements](#structure-accessibility-requirements). Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to true (passed) or false (failed). This translation of results is part of [rule aggregation](#rule-aggregation)
+<div class=note>
+  In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
```

Comment:
This is already described in the definition for outcome and can be removed from here.